### ISSUE TITLE
Load CSV as INT in yaml

### ISSUE THREAD

**Issue by songpr** [shoreditch-ops/artillery#563]
Cannot load INT from CSV to YAML
YAML

```yaml
config:
  target: 'http://localhost:8888'
  phases:
    - duration: 1
      arrivalRate: 20
  variables:
    url:
      -  "/m2019mbot"
    id:
      - 1
      - 2
  payload:
    path: "fb-messages.csv"
    fields:
      - "entry.id"
      - "entry.time"
      - "sender.id"
      - "recipient.id"
      - "timestamp"
      - "message.mid"
      - "message.seq"
      - "message.text"
scenarios:
  - flow:
    - post: #test subscription of facebook
        url: "{{url}}"
        json:
          object: "page"
          entry:
          - id: "{{id}}"
            time: "{{entry.time}}"
            messaging:
            - sender:
                id: "{{sender.id}}"
              recipient:
                id: "{{recipient.id}}"
              timestamp: "{{timestamp}}"
              message:
                mid: "{{message.mid}}"
                seq: "{{message.seq}}"
                text: "{{message.text}}"
```

fb-messages.csv

```csv
entry.id,entry.time,sender.id,recipient.id,timestamp,message.mid,message.seq,message.txt
"232905073499898",1541667978516,"1183939548360937","232905073499898",1541667977115,"XfIjM0vNh4988miZ5sRnRMFYocNwSNLbpkTjrTAMXsW3TI7466b6F-uqKs3pDJacPrDuei3B5imN3y7cx9cFIg",265796,"Hello"
"232905073499898",1541667978516,"1183939548360937","232905073499898",1541667977116,"XfIjM0vNh4988miZ5sRnRMFYocNwSNLbpkTjrTAMXsW3TI7466b6F-uqKs3pDJacPrDuei3B5imN3y7cx9cFIg",265797,"Hello1"
"232905073499898",1541667978516,"1183939548360937","232905073499898",1541667977117,"XfIjM0vNh4988miZ5sRnRMFYocNwSNLbpkTjrTAMXsW3TI7466b6F-uqKs3pDJacPrDuei3B5imN3y7cx9cFIg",265798,"Hello2"
"232905073499898",1541667978516,"1183939548360937","232905073499898",1541667977118,"XfIjM0vNh4988miZ5sRnRMFYocNwSNLbpkTjrTAMXsW3TI7466b6F-uqKs3pDJacPrDuei3B5imN3y7cx9cFIg",265799,"Hello3"
"232905073499898",1541667978516,"1183939548360937","232905073499898",1541667977119,"XfIjM0vNh4988miZ5sRnRMFYocNwSNLbpkTjrTAMXsW3TI7466b6F-uqKs3pDJacPrDuei3B5imN3y7cx9cFIg",265900,"Hello4"
```

Output 

```json
{"object":"page","entry":[{"id":2,"time":"1541667978516","messaging":[{"sender":{"id":"1183939548360937"},"recipient":{"id":"232905073499898"},"timestamp":"1541667977115","message":{"mid":"XfIjM0vNh4988miZ5sRnRMFYocNwSNLbpkTjrTAMXsW3TI7466b6F-uqKs3pDJacPrDuei3B5imN3y7cx9cFIg","seq":"265796","text":"Hello"}}]}]}
```

But I want entry.time  to be Int How?

**Comment by songpr** [comment]
@hassy  I see you close this #250 but there is no solution for load int variable using CSV file yet.

Could you please advice?

**Comment by hassy** [comment]
Thanks for the detail @songpr! I edited your original comment to fix up the formatting. What version of Artillery are you using? (`artillery -V`)

**Comment by songpr** [comment]
@hassy 1.6.0-25

I can achieve what I want by using function. But I think it will be easier to support via CSV natively.

**Comment by hassy** [comment]
This should be fixed in #580 - Artillery will now automatically try to cast fields to native types (e.g. numbers).
